tests/special_tests.sh

```diff
@@ -15,6 +15,7 @@
 set -e
 export PL_RUNNING_SPECIAL_TESTS=1
 DEFAULTS="-m coverage run --source pytorch_lightning -a -m pytest --verbose --capture=no"
+python ${DEFAULTS} tests/accelerator/test_accelerator_connector.py::test_accelerator_choice_ddp_cpu_custom_plugin
```

Comment:
@awaelchli Why does it go in special tests?

Reply:
I'm trying to figure that out... Ci fails and I dont know why. Locally all fine. So I thought maybe the ddp test needs to run as special.

Reply:
I'll get back on it today